bin/bundle/deploy/941/Connector-RALLY/deploy.info

```diff
@@ -0,0 +1,11 @@
+# -----------------------------------------------------------------------------
+# HPE PPM Integration Connector Rally v2.0
+# -----------------------------------------------------------------------------
+
+DESCRIPTION="HPE PPM Integration Connector Rally v2.0"
```

Comment:
Remove HPE from all the labels. 

HP Software will be sold to "microfocus" in June, and I don't want to have to update all labels then :)